.gitignore

```diff
@@ -19,3 +19,4 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 /tmp
+mix.lock
```

Comment:
Please don't gitignore this, it makes it incredibly hard to have reproducibility between developers on the project.

Reply:
Yeah, this is a really important file to have committed.